README.md

```diff
@@ -1 +1,90 @@
-# service_camp_capstone
\ No newline at end of file
+# service_camp_capstone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# portfolio
+
+> I create the nav and headline in my Capstone project, using HTML, CSS(flexbox), Linters.
+> I add the headlines, the projects cards, and also the about page.
+> I created the desktop version using media queries.
+> I Deployed my website using GitHub pages
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML and CSS
+- Media Query
+- Grid Layouts and Flexbox
+- Positioning
+- Technologies: Git and GitHub
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```

Comment:
- [ ] Please delete this instruction. This instruction is provided for you to know what you should include here ✔️

Reply:
done